cinch/playbooks/jenkins_restart_utility.yml

```diff
@@ -0,0 +1,139 @@
+# This playbook aims to gather diagnostics data in the event of
+# trouble with Jenkins, and then restarts Jenkins to keep operations running.
+# By default, only the diagnostics data is collected, but the Jenkins service
+# restart can be enabled by setting the 'restart_jenkins' variable to true:
+
+# Example:
+# ansible-playbook -u root --private-key /path/to/key
+# -i '10.0.183.169', jenkins_restart_utility.yml -e "restart_jenkins=true"
+
+- hosts: all
+  vars:
+    jenkins_pid_path: '/run/jenkins.pid'
```

Comment:
The same goes for the other many variables that are given default values inline. Does the value of "vars" here mess with variable search order the way that the vars directory in a role does? Maybe we could clean up the use of filters if we had default values here?